core/server/master/src/main/java/alluxio/master/meta/AlluxioMasterRestServiceHandler.java

```diff
@@ -839,6 +840,27 @@ public Response getWebUIWorkers() {
     }, ServerConfiguration.global());
   }
 
+  /**
+   * @param ufs the ufs uri encoded by {@link MetricsSystem#escape(AlluxioURI)}
+   * @return whether the ufs uri is a mount point
+   */
+  @VisibleForTesting
+  boolean isMounted(String ufs) {
+    if (ufs.endsWith("/")) {
```

Comment:
nit: add this to PathUtils and use AlluxioURI.SEPARATOR if a similar function does not exist?

Reply:
done